docs/_templates/layout.html

```diff
@@ -8,7 +8,7 @@
     {% if pagename in ["production/postgresql", "production/video-calls"] and release.endswith('+git') %}
     {#
     # This page doesn't exist in the stable documentation yet.
-    # This temporary workaround prevents CircleCI failure and should be removed after the next release.
+    # This temporary workaround prevents GitHub Actions failure and should be removed after the next release.
```

Comment:
I think this sort of notice should probably just say "prevents test failures ..." -- we don't need to name the branch of CI service we're using in this sort of comment.